### PR TITLE
📝 Added missing documentation

### PR DESCRIPTION
**Pull request by giselleghadyani**
Added documentation files for remaining exports.

### CODE REVIEW

File: api-reference/field-component.md [hunk 1]
```diff
@@ -56,9 +56,26 @@ const FieldExample = () => (
...
+      <th style="text-align:left">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="text-align:left"><code>children</code>
+      </td>
+      <td style="text-align:left">
+        <p>Node</p>
+        <p>A single component.</p>
```
Comment (Sawtaytoes) [inline api-reference/field-component.md:74]: The second one of these should be wrapped in `<em>`.

File: api-reference/submitfield-component.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# &lt;SubmitField /&gt;
+
+
+
```
Comment (Sawtaytoes) [inline api-reference/submitfield-component.md:4]: Unnecessary newlines?

**Comment by Sawtaytoes** [comment]
Thanks for all your hard work! Improving the docs will be a huge help!

File: api-reference/submitfield-component.md (hunk 2 above)
Comment (giselleghadyani) [reply api-reference/submitfield-component.md:4]: Copy/pasta. This is in other files as well, I think GitBook does that.

File: api-reference/field-component.md (hunk 1 above)
Comment (giselleghadyani) [reply api-reference/field-component.md]: 847b9e06cb75bdce733e3216442794e03167b340

**Pull request merged**